test/postControllerTest.js

```diff
@@ -0,0 +1,148 @@
+/* global it, describe */
```

Comment:
You shouldn't need this anymore since you added mocha to your eslintrc.

You _should_ also remove this line from your postServiceTest.js... If that feels weird, it's because adding mocha to eslintrc _should_ be a separate PR, and adding it here is scope creep.

This makes things difficult for everyone: For you since you can't address these features separately (ie you decide you want only 1 feature), and for reviewers since they have to figure out what feature each change belongs to.